package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "bb-practicum-team-1-back",
+  "name": "Beauty Finder GG-PRAC-TEAM5",
   "version": "1.0.0",
-  "description": "Back-End Repo for Team 1 of Baboon/Bald Eagle Practicum",
+  "description": "Back-End Repo for GG Team 5",
```

Comment:
same here
"Back-End Repo for Team 5 of Grackle/Giraffe",

Reply:
Done